store/postgres/migrations/000008_create_comments_table.up.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE comments (
+  id serial PRIMARY KEY,
+  discussion_id serial NOT NULL REFERENCES discussions(id) ON DELETE CASCADE,
+  body text NOT NULL,
+  owner uuid NOT NULL REFERENCES users(id) ON DELETE CASCADE,
```

Comment:
will it be better to name it `user_id` instead in the DB?

Reply:
do you mean like this?
```suggestion
  user_id uuid NOT NULL REFERENCES users(id) ON DELETE CASCADE,
```
IMO `user_id` is not self-explanatory, it could be anything. I am using owner because we are consistently using `owner` for all resources.

Reply:
no major issue actually, just used to convention having FK as `{singular_table_name}_id`.
we can keep owner, should be fine.